### DISCUSSION TITLE
截图是用的啥工具呀？挺好看

### DISCUSSION THREAD

**Discussion by ladypiapia** [lobehub/lobehub#636] [General]
![image](https://github.com/lobehub/lobe-chat/assets/36269124/e9124538-6bc4-4b3a-a8a4-6417f26f509a)
这个截图中的引导标识是通过工具实现的还是自己手动另外单独处理的呀？

**Comment by arvinxx** [comment]
在设计稿里画的

**Reply by ladypiapia** [reply]
OK，感谢回复~整个项目做得太好了！颜值也很高，赞！
